magento/adobe-stock-integration
# Provide meaningful naming for saved image files

### Description (*)
Provided meaningful naming for saved image files
Generated based on title

### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#323: Move authentication to AdobeIMS module
2. magento/adobe-stock-integration#328: User sets the image name

### Manual testing scenarios (*)

1. Save preview image
2. Verify image and image name.
